x-pack/plugins/security_solution/cypress/e2e/exceptions/shared_exception_lists_management/shared_exception_list_page/duplicate_lists.cy.ts

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { createRule } from '../../../../tasks/api_calls/rules';
+import { getExceptionList } from '../../../../objects/exception';
+import { assertNumberOfExceptionItemsExists } from '../../../../tasks/exceptions';
+import {
+  assertExceptionListsExists,
+  duplicateSharedExceptionListFromListsManagementPageByListId,
+  findSharedExceptionListItemsByName,
+  waitForExceptionsTableToBeLoaded,
+} from '../../../../tasks/exceptions_table';
+import { login, visitWithoutDateRange } from '../../../../tasks/login';
+import { EXCEPTIONS_URL } from '../../../../urls/navigation';
+import {
+  createExceptionList,
+  createExceptionListItem,
+} from '../../../../tasks/api_calls/exceptions';
+import { esArchiverResetKibana } from '../../../../tasks/es_archiver';
+import { getNewRule } from '../../../../objects/rule';
+
+const expiredDate = new Date(Date.now() - 1000000).toISOString();
+const futureDate = new Date(Date.now() + 1000000).toISOString();
+const EXCEPTION_LIST_NAME = 'My test list';
+const EXCEPTION_LIST_TO_DUPLICATE_NAME = 'A test list 2';
+const EXCEPTION_LIST_ITEM_NAME = 'Sample Exception List Item 1';
+const EXCEPTION_LIST_ITEM_NAME_2 = 'Sample Exception List Item 2';
+
+const getExceptionList1 = () => ({
+  ...getExceptionList(),
+  name: EXCEPTION_LIST_NAME,
+  list_id: 'exception_list_1',
+});
+const getExceptionList2 = () => ({
+  ...getExceptionList(),
+  name: EXCEPTION_LIST_TO_DUPLICATE_NAME,
+  list_id: 'exception_list_2',
+});
+
+describe('Duplicate List', () => {
+  before(() => {
+    esArchiverResetKibana();
```

Comment:
Please, check if the use of this method is extremely necessary. Thanks.

Reply:
This test needs the env to be reset before it runs